cmd/catalog.go

```diff
@@ -450,7 +450,20 @@ func (c *CatalogOperator) forEachTemplate(f func(item *catalog.Template) error)
 		return err
 	}
 
-	for _, item := range collection.Data {
+	collectiondata := collection.Data
```

Comment:
this code seems to repeat in several places. Any way we can extract it to some helper method, and call from there?

Reply:
Looked at the code more, seems like implementing it in generic manner using interfaces would be even more cumbersome.